packages/blocto-sdk/src/providers/ethereum.ts

```diff
@@ -162,6 +162,14 @@ export default class EthereumProvider
     }
   }
 
+  #extractParams(params: Array<any>): Array<any> {
+    return params.map(param => {
```

Comment:
Overall it's okay, but I think better separate batch handler from `handleSendTransaction`.
In your code, it will look through all element in array and has O(n) complexity. 
But actually, it can be separated from beginning since we know it's method will be `blocto_sendBatchTransaction`.

Reply:
And since this function will only used here and it doesn't has dependency in class, please either move it into batch transaction function or move out of class.